docs/testing-strategy.md

```diff
@@ -0,0 +1,152 @@
+# Testing Strategy
+
+We aim for the Content Publisher to have a test suite that provides developers
+with value. We want the application to be fast to test; for the tests to catch
+breaking behavioural changes; and for it to be easy for developers to know
+where to add tests. The purpose of this document is
```

Comment:
I don't think we're explicitly aiming for maximum test coverage (that'd be 100%) as that seems to often be an anti-pattern to be too worried about that number.

Reply:
Hmm true, I think my wording was a lil dodgy here. I was more meaning that all of the features we build are thoroughly tested rather than worrying that every line of code has been hit. However, we're apparently hitting 97.74% code coverage according to RSpec's report which sounds nice! 😄